xbmc/interfaces/legacy/Dialog.h

```diff
@@ -746,112 +743,6 @@ constexpr int ALPHANUM_HIDE_INPUT{2};
 
     //@}
 
-    ///
-    /// \defgroup python_DialogBusy DialogBusy
-    /// \ingroup python_xbmcgui
-    /// @{
-    /// @brief <b>Kodi's busy dialog class</b>
-    ///
-    ///-----------------------------------------------------------------------
-    /// @python_v18 removed, usage results in nop!
```

Comment:
All methods should be added to `python_removed_functions` see: https://github.com/xbmc/xbmc/blob/12ec6426dd5882e6b0a53ec0900c599714539bf8/xbmc/addons/kodi-addon-dev-kit/doxygen/Modules/modules_python.dox#L150-L160